documentation/source/quickstart.rst

```diff
@@ -18,104 +18,92 @@ Cocotb has the following requirements:
 
 .. versionchanged:: 1.4 Dropped Python 2 support
 
-.. _installation_via_pip:
+.. note:: In order to use a 32bit simulator you need to use a 32bit version of Python.
 
-Installation via PIP
---------------------
+Installation with conda (all OS)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. versionadded:: 1.2
+`Conda <https://conda.io/>`_ is an open-source package and environment management system that runs on Windows, macOS and Linux.
 
-Cocotb can be installed by running
+Download and install `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ from https://conda.io/ for your OS and architecture.
 
-.. code-block:: bash
+You will also need to install a compiler (GCC or Clang) and GNU Make.
+
+On Windows you can easily do this with conda:
 
-    pip3 install cocotb
+.. code-block:: bash
 
-For user local installation follow the
-`pip User Guide <https://pip.pypa.io/en/stable/user_guide/#user-installs/>`_.
+    conda install -c msys2 m2-base m2-make m2w64-toolchain libpython
 
-To install the development version of cocotb:
+Native Installation for Debian/Ubuntu-based Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: bash
 
-    git clone https://github.com/cocotb/cocotb
-    pip3 install -e ./cocotb
+    sudo apt-get install make gcc g++ swig python3 python3-dev python3-pip
 
-.. note::
+Native Installation for Red Hat-based Systems
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    After installation, you should be able to execute ``cocotb-config``.
-    If it is not found, you need to append its location to the ``PATH`` environment variable.
-    This may happen when you use the ``--user`` option to ``pip``, in which case the location is documented :ref:`here <python:inst-alt-install-user>`.
+.. code-block:: bash
 
-Native Linux Installation
--------------------------
+    sudo yum install make gcc gcc-c++ libstdc++-devel swig python3 python3-devel python3-pip
 
-The following instructions will allow building of the cocotb libraries
-for use with a 64-bit native simulator.
+macOS Packages
+^^^^^^^^^^^^^^
 
-If a 32-bit simulator is being used then additional steps are needed, please see
-`our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
+You need a few packages installed to get cocotb running on macOS.
+Installing a package manager really helps things out here.
 
-Debian/Ubuntu-based
-^^^^^^^^^^^^^^^^^^^
+`Brew <https://brew.sh/>`_ seems to be the most popular, so we'll assume you have that installed.
 
 .. code-block:: bash
 
-    sudo apt-get install git make gcc g++ swig python-dev
+    brew install python icarus-verilog gtkwave
 
-Red Hat-based
-^^^^^^^^^^^^^
+.. seealso::
 
-.. code-block:: bash
+   For more installation options (also for 32-bit simulators) please see `our Wiki <https://github.com/cocotb/cocotb/wiki/Tier-2-Setup-Instructions>`_.
```

Comment:
I think this should be below the `Simulator Installation` heading

Reply:
I have changed this a bit to remove `simulators`  since this was the intention. Feel free to change.